bocas/artifacts/artifact.py

```diff
@@ -1,3 +1,15 @@
+from .yaml_utils import parse_yaml_node
```

Comment:
can we import `from bocas.artifacts.yaml_utils import parse_yaml_node` instead of `.yaml_utils`?

Reply:
Good catch. Adjusted